snapcraft/plugins/catkin.py

```diff
@@ -161,22 +160,23 @@ def _find_package_deps(self):
         if self.package_deps_found:
             return
 
```

Comment:
A comment here would be nice. Like: 
some catkin packages have a src where they put all the package code, some just put the code in the root of the repository.